src/layman/common/output_srs_list_test.py

```diff
@@ -43,19 +44,21 @@ def test_custom_srs_list(ensure_layer):
     layer_qgis1 = 'test_custom_srs_list_qgis_layer1'
     layer_qgis2 = 'test_custom_srs_list_qgis_layer2'
     source_style_file_path = 'sample/style/small_layer.qml'
-    assert settings.LAYMAN_OUTPUT_SRS_LIST != OUTPUT_SRS_LIST
+    output_crs_list = {f'EPSG:{int(srid)}' for srid in OUTPUT_SRS_LIST}
```

Comment:
Is int() used intentionally to raise exception if there is accidently string in OUTPUT_SRS_LIST? If not, please remove it.